packages/browser-wallet-api-helpers/package.json

```diff
@@ -4,8 +4,17 @@
     "license": "Apache-2.0",
     "packageManager": "yarn@3.2.0",
     "main": "lib/index.js",
-    "browser": "lib/concordiumHelpers.min.js",
+    "browser": "./lib/concordium.min.js",
     "types": "lib/index.d.ts",
+    "cdn": "./lib/concordium.min.js",
```

Comment:
How is this supposed to work, the built file is still named `concordiumHelpers.min.js`?

Reply:
You're right.. Don't know where I got this from 🤷